Unable to compile packages

```
yarn run v1.22.17
error Command "compile" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/release-code.yaml
+++ b/.github/workflows/release-code.yaml
@@ -38,11 +38,11 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-yarn-
 
-      - name: Install frontend dependencies
+      - name: Install dependencies
         run: yarn --frozen-lockfile
 
-      - name: Compile the packages
-        run: yarn compile
+      - name: Build the packages
+        run: yarn build
 
       # We don't use the default token since it won't call the release trigger
       # It is an intentional limitation from GitHub

